setup.py

```diff
@@ -258,7 +258,7 @@ def get_exts_for(name):
         "pathlib2; python_version<'3.5'",
         "jsonschema",
         "xmltodict>=0.12",
-        "backport_ipaddress; python_version=='2.7'",
+        "ipaddress",
```

Comment:
no `; python_version=='2.7'` needed?

Reply:
Nope, works seamlessly on 3.6+ and it's actually also needed for 3.5 too!